lib/teleterm/api/uri/uri_test.go

```diff
@@ -52,34 +52,32 @@ func TestString(t *testing.T) {
 	}
 }
 
-func TestParseClusterURI(t *testing.T) {
+func TestGetClusterURI(t *testing.T) {
 	tests := []struct {
-		in  string
+		in  uri.ResourceURI
 		out uri.ResourceURI
 	}{
 		{
-			"/clusters/cluster.sh",
+			uri.New("/clusters/cluster.sh"),
```

Comment:
What's the advantage of using `uri.New` followed by serializing the URI to a string in the test over passing a string as the `in` field?